test/integrations/destinations/google_adwords_offline_conversions/dataDelivery/business.ts

```diff
@@ -0,0 +1,728 @@
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: generateProxyV0Payload({
+          headers: headers.header2,
+          params: params.param3,
+          JSON: validRequestPayload2,
+          endpoint:
+            'https://googleads.googleapis.com/v14/customers/1234567891:uploadClickConversions',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            destinationResponse: {
+              response: [
+                {
+                  adjustmentDateTime: '2021-01-01 12:32:45-08:00',
+                  adjustmentType: 'ENHANCEMENT',
+                  conversionAction: 'customers/1234567891/conversionActions/874224905',
+                  gclidDateTimePair: {
+                    conversionDateTime: '2021-01-01 12:32:45-08:00',
+                    gclid: '1234',
+                  },
+                  orderId: '12345',
+                },
+              ],
+              status: 200,
+            },
+            message:
+              '[Google Ads Offline Conversions Response Handler] - Request processed successfully',
+            status: 200,
+          },
+        },
+      },
+    },
+  },
+];
+
+export const testScenariosForV1API = [
+  {
+    id: 'gaoc_v1_scenario_1',
+    name: 'google_adwords_offline_conversions',
+    description:
+      '[Proxy v1 API] :: Test for invalid argument - where the destination responds with 400 with invalid argument error',
+    successCriteria: 'Should return 400 with error with destination response',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers: headers.header1,
+            params: params.param1,
+            JSON: invalidArgumentRequestPayload,
+            endpoint:
+              'https://googleads.googleapis.com/v14/customers/11122233331/offlineUserDataJobs',
+          },
+          metadata,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message:
+              '[Google Ads Offline Conversions]:: Request contains an invalid argument. during google_ads_offline_store_conversions Add Conversion',
+            response: [
+              {
+                error:
+                  '[Google Ads Offline Conversions]:: Request contains an invalid argument. during google_ads_offline_store_conversions Add Conversion',
+                metadata: {
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  jobId: 1,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                },
```

Comment:
We should use a variable rather than writing so much info

Reply:
addressed